spec/dealership_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'rspec'
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe Dealership do
```

Comment:
You may want to look into `before(:each)` so you don't need to redefine 

```
dealership = Dealership.new("Acme Auto", "123 Main Street")
car_1 = Car.new("Ford Mustang", 1500, 36)
car_2 = Car.new("Toyota Prius", 1000, 48)
```

etc. in every test